ui/app/components/send/send-content/send-gas-row/send-gas-row.container.js

```diff
@@ -42,7 +42,7 @@ function mapStateToProps (state) {
   const balance = getCurrentEthBalance(state)
 
   const insufficientBalance = !isBalanceSufficient({
-    amount: getSendAmount(state),
+    amount: getSelectedToken(state) ? '0x0' : getSendAmount(state),
```

Comment:
Should we be casting the result to a boolean to be more explicit?

```suggestion
    amount: Boolean(getSelectedToken(state)) ? '0x0' : getSendAmount(state),
```